package NYT.NCellMaster.NProto;

////////////////////////////////////////////////////////////////////////////////

message TReqRegisterSecondaryMaster
{
    required int32 cell_tag = 1;
}

message TReqRegisterAtPrimaryMaster
{ }

////////////////////////////////////////////////////////////////////////////////
